internal/server/statvar/golden/get_place_stat_vars/IG_idn.json

```diff
@@ -0,0 +1,3816 @@
+        "WHO/pncall3_Quintile2",
+        "WHO/pncall3_Quintile3",
+        "WHO/pncall3_Quintile4",
+        "WHO/pncall3_Quintile5",
+        "WHO/pncall3_Rural",
+        "WHO/pncall3_SecondaryOrHigher",
+        "WHO/pncall3_Urban",
+        "WHO/pncall5_Female",
+        "WHO/pncall5_Male",
+        "WHO/pncall5_NoSchoolingCompleted",
+        "WHO/pncall5_Primary",
+        "WHO/pncall5_Quintile1",
+        "WHO/pncall5_Quintile2",
+        "WHO/pncall5_Quintile3",
+        "WHO/pncall5_Quintile4",
+        "WHO/pncall5_Quintile5",
+        "WHO/pncall5_Rural",
+        "WHO/pncall5_SecondaryOrHigher",
+        "WHO/pncall5_Urban",
+        "WHO/pnchome3_Female",
+        "WHO/pnchome3_Male",
+        "WHO/pnchome3_NoSchoolingCompleted",
+        "WHO/pnchome3_Primary",
+        "WHO/pnchome3_Quintile1",
+        "WHO/pnchome3_Quintile2",
+        "WHO/pnchome3_Quintile3",
+        "WHO/pnchome3_Quintile4",
+        "WHO/pnchome3_Quintile5",
+        "WHO/pnchome3_Rural",
+        "WHO/pnchome3_SecondaryOrHigher",
+        "WHO/pnchome3_Urban",
+        "WHO/pnchome5_Female",
+        "WHO/pnchome5_Male",
+        "WHO/pnchome5_NoSchoolingCompleted",
+        "WHO/pnchome5_Primary",
+        "WHO/pnchome5_Quintile1",
+        "WHO/pnchome5_Quintile2",
+        "WHO/pnchome5_Quintile3",
+        "WHO/pnchome5_Quintile4",
+        "WHO/pnchome5_Quintile5",
+        "WHO/pnchome5_Rural",
+        "WHO/pnchome5_SecondaryOrHigher",
+        "WHO/pnchome5_Urban",
+        "WHO/poliov_Female",
+        "WHO/poliov_Male",
+        "WHO/poliov_NoSchoolingCompleted",
+        "WHO/poliov_Primary",
+        "WHO/poliov_Quintile1",
+        "WHO/poliov_Quintile2",
+        "WHO/poliov_Quintile3",
+        "WHO/poliov_Quintile4",
+        "WHO/poliov_Quintile5",
+        "WHO/poliov_Rural",
+        "WHO/poliov_SecondaryOrHigher",
+        "WHO/poliov_Urban",
+        "WHO/vbcg_Female",
+        "WHO/vbcg_Male",
+        "WHO/vbcg_NoSchoolingCompleted",
+        "WHO/vbcg_Primary",
+        "WHO/vbcg_Quintile1",
+        "WHO/vbcg_Quintile2",
+        "WHO/vbcg_Quintile3",
+        "WHO/vbcg_Quintile4",
+        "WHO/vbcg_Quintile5",
+        "WHO/vbcg_SecondaryOrHigher",
+        "WHO/vita_Decile1",
+        "WHO/vita_Decile10",
+        "WHO/vita_Decile2",
+        "WHO/vita_Decile3",
+        "WHO/vita_Decile4",
+        "WHO/vita_Decile5",
+        "WHO/vita_Decile6",
+        "WHO/vita_Decile7",
+        "WHO/vita_Decile8",
+        "WHO/vita_Decile9",
+        "WHO/vita_Female",
+        "WHO/vita_Male",
+        "WHO/vita_NoSchoolingCompleted",
+        "WHO/vita_Primary",
+        "WHO/vita_Quintile1",
+        "WHO/vita_Quintile2",
+        "WHO/vita_Quintile3",
+        "WHO/vita_Quintile4",
+        "WHO/vita_Quintile5",
+        "WHO/vita_SecondaryOrHigher",
+        "dc/0ny9m7k0lzmn5",
+        "dc/7fmnve6yyc508",
+        "dc/n9rqwq8d3lcx3",
+        "dc/p49ghylpwz93f",
+        "dc/qdbhwwr88yhl7",
+        "dc/tbbjnf9bk96yh",
```

Comment:
@pradh This is the same as Covid19MobilityTrend_Workplace. Is there a preference on the human curated stat var?

Reply:
Wait this is not expected... if there is a human curated SV, that should be preferred... let me see